auth-service/src/test/kotlin/uk/gov/communities/delta/security/RateLimitingTest.kt

```diff
@@ -140,11 +137,57 @@ class RateLimitingTest {
         for (i in 1..rateLimitValue) {
             assertSuccessfulGetRequest(forwardFor1, "/delta/set-password")
         }
-        assertBlockedGetLoginRequest(forwardFor1, "/delta/set-password", "<title>Delta | Set password</title>")
+        assertBlockedGetRequest(forwardFor1, "/delta/set-password", "<title>Delta | Password</title>")
         assertSuccessfulGetRequest(forwardFor2, "/delta/set-password")
         verify(exactly = 1) { rateLimitSetPasswordCounter.increment(1.0) }
     }
 
+    @Test
+    fun testResetPasswordPageRateLimit() = testSuspend {
+        val forwardedFor = "5.6.7.8, 1.2.3.4, 5.6.7.8"
+        for (i in 1..rateLimitValue) {
+            assertSuccessfulGetRequest(forwardedFor, "/delta/reset-password")
+        }
+        assertBlockedGetRequest(forwardedFor, "/delta/reset-password", "<title>Delta | Password</title>")
+        verify(exactly = 1) { rateLimitResetPasswordCounter.increment(1.0) }
+    }
+
+    @Test
+    fun testResetPasswordPageRateLimitPerUser() = testSuspend {
```

Comment:
We probably don't need all four tests for each endpoint, doesn't do any harm, but one for each endpoint and the other three just for one of them would be fine